Makefile

```diff
@@ -3,6 +3,8 @@ SHELL := /bin/bash
 
 .PHONY : clean build test publish
```

Comment:
Should update the PHONY targets to include release (and some other missing targets).